core/01_core.js

```diff
@@ -127,6 +127,15 @@
     errorMap[className] = errorBuilder;
   }
 
+  function buildCustomError(className, message) {
+    const error = errorMap[className]?.(message);
+    // Strip buildCustomError() calls from stack trace
+    if (typeof error == "object") {
+      ErrorCaptureStackTrace(error, buildCustomError);
+    }
+    return error;
+  }
+
   function unwrapOpResult(res) {
```

Comment:
You should now be able to remove the `unwrapOpResult` function, yes?

Reply:
It's still used for async ops

Reply:
Ah, mb